client/index.html

```diff
@@ -9,22 +9,31 @@
 
 <body>
 
-	<div id="backstage"></div>
-
-	<div id="header">Data Set.csv</div>
-	
-	<div class="silky-ribbon">
-	</div>
-	
-	<div id="main-view">
-		<div id="main-table">This is the main table</div>
-		<div id="results"><h1>Results</h1></div>
-	</div>
-    
+    <div id="backstage"></div>
+
+    <div id="header">Data Set.csv</div>
+
+    <div class="silky-ribbon">
+    </div>
+
+    <div id="main-view">
```

Comment:
oh right, github assumes you're making comments about the preceding code, rather than the latter, and only puts the code before into the conversation. this is how i imagined you would see it:

<img width="789" alt="screen shot 2016-02-16 at 10 28 40" src="https://cloud.githubusercontent.com/assets/3240247/13063040/1cad7f12-d498-11e5-8e26-24b08c7659cc.png">